enforcer/datapath/datapath.go

```diff
@@ -47,10 +46,11 @@ type Datapath struct {
 
 	// Internal structures and caches
 	// Key=ContextId Value=ContainerIP
-	contextTracker cache.DataStore
-	puFromIP       cache.DataStore
-	puFromMark     cache.DataStore
-	puFromPort     cache.DataStore
+	contextTracker   cache.DataStore
```

Comment:
As a cleanup can you rename contextTracker to ipFromContextID (Feel free to do it in a different PR or here while making some other changes). Also, if you do this the comment above this becomes useless and can be removed.